### ISSUE TITLE
UX confusions as mentor filing a new project

### ISSUE THREAD

**Issue by danvet** [sagesharp/outreachy-django-wagtail#146]
While it's all still fresh. I figured I'll file one big issue, but happy to split it up if you want me to. For context the project I'm mentoring for:

https://www.outreachy.org/communities/cfp/linux-kernel/project/dri-devel-aka-kernel-gpu-subsystem/

- I can't find projects from the webpage. I tried clicking around quite a bit, and the only thing I ever found was https://www.outreachy.org/apply/project-selection/ but that doesn't have the link to the kernel org. This is when navigating from the outreachy.org main page, and logged in as mentor. My approach now is to find a mail with a link to my project, and then there's a nice breadcrumb at the top that lets me navigate to https://www.outreachy.org/communities/cfp/ But I can't find that any other way at all. For context: I really wanted to find that to look at other projects so I have some examples what the various fields are used for. I think if the buttons at the top would have submenus for the most important options that would help a lot in navigating the site (the links are probably all somewhere in the wall of text, but that makes them really hard to discover).

- The proprietary software/approved license flow is a bit confusing to me visually. Maybe a yes/no selection box and hiding the big text field as long as no selection has been made yet (or as long as it's not needed) would help, only showing the textfield when the the answers are No.

- Some visual indicators that the project creation form has 3 pages would be great. I freaked out on the second page, assuming the form will lose my stuff and just madly clicked through them all until the end :-) Then edited afterwards. Similar, when editing the project it'd be great if I could jump to a specific page and only edit that (maybe 3 edit buttons for that?).

- For the communication channels and intern requirements pages: Some visual indication (cards/folders/boxes) what belongs together. Especially the Delete button looks very dangerous ("Nope not going to click that, it might delete my project"). Also if I could collapse them for a better overview (the page is rather long already with the default 3 items) while editing - avoids entering IRC twice and stuff like that.

**Comment by danvet** [comment]
Another one: The CFP page for a project/community vs. the page that's shown to students/applicants:
https://www.outreachy.org/communities/cfp/linux-kernel/ vs https://www.outreachy.org/2018-may-august/communities/linux-kernel/

It took me a while to realize it's not the same thing and that there's 2 completely different links and that I picked the wrong one from my browser history. For context, I wanted to fish out the kernel's intro section to answer a question of an applicant, but on the CFP page it's not shown and I got confused until I stumbled over the other page.
